src/adapters/scene.js

```diff
@@ -145,10 +147,29 @@ Scene.prototype.initMapBox = function() {
       }, this.DOUBLE_TAP_DELAY_MS);
     });
 
+    // Long touch on mobile (500ms)
+    // Custom implementation because the contextmenu event isn't supported by MapBox.
+    // Long touch is initiated with a touchstart event
+    // It is canceled if many fingers touch the map, or if one of these events is initiated:
+    // touchmove, touchcancel, touchend, pointerdrag, pointermove, moveend, gesturestart, gesturechange, gestureend
```

Comment:
We don't need to list every events in the comments here, they already very readable in the code (see my other comment).
BTW, did you establish this list yourself? Or if it's inspired by a documented example, it would be more interesting to have the link to it in these comments.